frontend/src/components/workspaces/Templates/TemplatesTableRowSettings/TemplatesTableRowSettings.tsx

```diff
@@ -0,0 +1,59 @@
+import { Menu, Dropdown } from 'antd';
+import Button from 'antd-button-color';
+import {
+  PlayCircleOutlined,
+  EditOutlined,
+  DeleteOutlined,
+  SettingOutlined,
+} from '@ant-design/icons';
+
+export interface ITemplatesTableRowSettingsProps {
+  id: string;
+  editTemplate: (id: string) => void;
+  deleteTemplate: (id: string) => void;
+}
+const TemplatesTableRowSettings = ({ ...props }) => {
+  const { id, editTemplate, deleteTemplate } = props;
+
+  return (
+    <Dropdown
+      overlay={
+        <Menu>
+          <Menu.Item
+            className="xs:hidden block"
+            key="1"
+            icon={<PlayCircleOutlined />}
+          >
+            Create
+          </Menu.Item>
+          <Menu.Item
+            key="1"
+            icon={<EditOutlined />}
+            onClick={() => editTemplate(id)}
+          >
+            Edit
+          </Menu.Item>
+          <Menu.Item
+            danger
+            key="2"
+            icon={<DeleteOutlined />}
+            onClick={() => deleteTemplate(id)}
+          >
+            Delete
+          </Menu.Item>
+        </Menu>
+      }
+      placement="bottomCenter"
+      trigger={['click']}
+    >
+      <Button
+        with="link"
+        type="lightdark"
+        size="large"
+        icon={<SettingOutlined style={{ fontSize: '22px' }} />}
```

Comment:
What do you think of using the 3-dots (`MoreOutlined`) instead of the `SettingsOutlined`?

Reply:
due it appears only in manager view and rappresents Template Setting as edit and delete function

Reply:
I think it could be a good idea to use 3-dots only on mobile version

Reply:
Changed applied [here](https://github.com/netgroup-polito/CrownLabs/pull/460/commits/77c8d7a732e466080c9e305ba922259e2ecdaa42)